### PR TITLE
PLEASE IGNORE  - Mig virt mach cli hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
@anarnold97 @bkhizgiy

MTV-428: https://issues.redhat.com/browse/MTV-428

Added option for new field insecureSkipVerify to 5.1 Migrating virtual machines from the command line.
See lines 38 and 46.

### CODE REVIEW

**Comment by anarnold97** [comment]
@HagayVider1 Looks ok to me. If it is ok, please do not forget to ask for a peer review
@bkhizgiy - Is this ok with you?

File: documentation/modules/rn-2.4.adoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Faster disk image migration without using virt-v2v
+
+IDisk images are not converted anymore using virt-v2v when migrating from RHV. This way, the migration is faster and not limited to guest operating systems that are supported by virt-v2v. link:https://github.com/kubev2v/forklift-controller/issues/403[(forklift-controller#403)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:19]: Please can we remove "anymore"?

Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:19]: I am not very keen on "... converted anymore" but it is personal

File: documentation/modules/rn-2.4.adoc [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+You must update the `StorageProfile` resource with `accessModes` and `volumeMode` for non-provisioner storage classes such as NFS. The documentation includes a link to the relevant procedure.
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.OCP console plugin
+
+This version of {project-full} has replaced the stand-alone UI with a UI that is integrated in the OCP Console. The new UI operates as an OCP Console plugin. It implements by default in version 2.4, disabling the old UI. You can enable the old UI by setting `feature_ui: true` in ForkliftController. link:https://issues.redhat.com/browse/MTV-427[(MTV-427)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:40]: Please could we change "It implements by default..." to "It is implemented by...". I think the verb tense is a bit awkward

File: documentation/modules/rn-2.4.adoc [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+.OpenStack migration
+
+ee new link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#rhv-prerequisites_mtv[prerequisites] for RHV and VMware migration, and link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[procedure] to migrate virtual machines. link:https://issues.redhat.com/browse/MTV-426[(MTV-426)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:44]: Please can you check "ee new link:" is correctly formatted?

File: documentation/modules/rn-2.4.adoc [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+.Skip certification option
+
+Skip certificate validation' option was added to VMware and RHV providers. If checked, the provider's certificate will not be validated and the UI will not ask for specifying a CA certificate.
+
+.Only third-party certificate required
+
+When defining a RHV provider that set with third-party certificate, only the third-party certificate need to be specified and MTV would retrieve the Manager CA certificate automatically.
```
Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:52]: "certificate need to be specified..." is a subject verb issue.
please, can we change to "certificate needs to be specified..."

Comment (anarnold97) [reply documentation/modules/rn-2.4.adoc:52]: Also, I would remove "MTV would retrieve..." and use "MTV retrieves...."

File: documentation/modules/rn-2.4.adoc [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+.Migration fails during precopy/cutover while a snapshot operation is executed on the source VM
+
+Some warm migrations from RHV might fail. When running a migration plan for warm migration of multiple VMs from RHV, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run. link:https://bugzilla.redhat.com/show_bug.cgi?id=2063531[(BZ#2063531)] link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+Warm migration from RHV fails if a snapshot operation is performed on the source VM. If the user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://bugzilla.redhat.com/show_bug.cgi?id=2057459[(BZ#2057459)]
+
+.Cannot schedule migrated VM with multiple disks to more than one storage classes of typw hostpath
```
Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:94]: should it be "type hostpath"?

File: documentation/modules/rn-2.4.adoc [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+== Resolved issues
+
+This release has the following resolved issue:
+
+.Map IDE to SATA +
+CNV does not support IDE disk interface, so the disks are mapped to SATA. link:https://issues.redhat.com/browse/MTV-345[(MTV-345)]
+
+.Improve invalid/conflicting VM name handling
+
+Users can automate VM renaming during mogration, to fit RFC 1123. link:https://issues.redhat.com/browse/MTV-212[(MTV-212)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.4.adoc:128]: Should it be "migration" not "mogration"?

File: documentation/modules/rn-2.4.adoc (hunk 2 above)
Comment (anarnold97) [reply documentation/modules/rn-2.4.adoc]: changed

File: documentation/modules/rn-2.4.adoc (hunk 5 above)
Comment (anarnold97) [reply documentation/modules/rn-2.4.adoc]: changed

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 7]
```diff
@@ -35,15 +35,17 @@ type: Opaque
...
     <engine_ca_certificate>
-  thumbprint: <vcenter_fingerprint> <4>
+  thumbprint: <vcenter_fingerprint> <5>
 EOF
 ----
 <1> Specify the vCenter user or the {rhv-short} {manager} user.
 <2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
-<4> VMware only: Specify the vCenter SHA-1 fingerprint.
+<3> Specify `<true>` to skip certificate verification, which proceeeds with an insecure migration. Specifying `<false>` verifies the certificate.
```
Comment (ahadas) [inline documentation/modules/migrating-virtual-machines-cli.adoc:46]:
```suggestion
<3> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration. Specifying `<false>` verifies the certificate.
```
and maybe also to emphasis what insecure migration means - that the transferred data is sent over an insecure connection and thus sensitive data might be exposed

Comment (bkhizgiy) [reply documentation/modules/migrating-virtual-machines-cli.adoc:46]: +1 to Arik comment, and maybe worth mention that the certificated is not needed if this value is set to true?

File: documentation/modules/rn-2.4.adoc (hunk 1 above)
Comment (HagayVider1) [reply documentation/modules/rn-2.4.adoc:19]: Changed to "_virt-v2v_ no longer converts IDisk images when migrating from RHV."

File: documentation/modules/rn-2.4.adoc (hunk 3 above)
Comment (HagayVider1) [reply documentation/modules/rn-2.4.adoc:44]: Corrected "ee" to "See"
Checked links in IntelliJ. They open the correct pages and section.

File: documentation/modules/rn-2.4.adoc (hunk 4 above)
Comment (HagayVider1) [reply documentation/modules/rn-2.4.adoc:52]: Corrected

**Comment by anarnold97** [comment]
Closed - obsolete

**Pull request closed without merging**